api/views.py

```diff
@@ -921,25 +921,25 @@ def update_app_status(request: HttpRequest) -> HttpResponse:
 
             if result == HandleUpdateStatusResponseCode.NO_ACTION:
                 return Response(
-                    "OK. NO_ACTION. No action performed. Possibly the event time is older \
-                    than the currently stored time.",
+                    f"OK. NO_ACTION. No action performed. Possibly the event time is older \
+                    than the currently stored time. {release=}, {new_status=}",
```

Comment:
I understand the comment but because the event listener passes in release I want to be clear what it actually is working on in respect to release vs subdomain.